aopy/analysis/behavior.py

```diff
@@ -146,6 +147,61 @@ def compute_path_length_per_trajectory(trajectory):
     path_length = np.sum(lengths)
     return path_length
 
+def compute_movement_stats(trajectory, target_position, rotation_axis=np.array([1, 0]), error_axis=1, 
+                           return_all_stats=False):
+    """
+    Computes movement statistics of a trajectory relative to a target position. 
+
+    Args:
+        trajectory (nt, 2): The trajectory coordinates for each point in time. 
+        target_position (2,): Target position coordinates.
+        rotation_axis ((2,) array, optional): The axis onto which movement will be
+            projected to calculate error. Defaults to np.array([1, 0]).
+        error_axis (int, optional): axis (after rotation) along which to compute the error 
+            statistics. Default 1.
```

Comment:
Having these both called axis but referring to two different types of axis is confusing. Maybe change 'rotation_axis' to 'rotation_vector'?